test/functional/ui/output_spec.lua

```diff
@@ -131,4 +139,91 @@ describe("shell command :!", function()
     ]])
     feed([[<CR>]])
   end)
+
+  describe('', function()
```

Comment:
describe("shell command :!", function()

Reply:
My mistake, it looked like the same indent level as line 80.  I would normally put it in its own scope but doesn't matter I guess.